### PR TITLE
Fixing OrchardCore.Localization dependencies

### PR DESCRIPTION
**Pull request by sebastienros**
These packages should be independent of any other OrchardCore project as they need to be usable from any ASP.NET application.

### CODE REVIEW

File: src/OrchardCore.Modules/OrchardCore.ContentFields/Views/HtmlField-Multiline.Edit.cshtml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 @model OrchardCore.ContentFields.ViewModels.EditHtmlFieldViewModel
-@using OrchardCore.ContentManagement.Metadata.Models
 @using OrchardCore.ContentFields.Settings
+@using OrchardCore.ContentManagement.Metadata.Models
```
Comment (hishamco) [inline src/OrchardCore.Modules/OrchardCore.ContentFields/Views/HtmlField-Multiline.Edit.cshtml]: Can we move the namespaces to `_ViewImports.cshtml`, while there are referenced a lot in many views?

File: src/OrchardCore.Modules/OrchardCore.ContentFields/Views/HtmlField-Multiline.Edit.cshtml [hunk 2]
```diff
@@ -1,6 +1,7 @@
...
+@using OrchardCore.ContentLocalization;
```
Comment (hishamco) [inline src/OrchardCore.Modules/OrchardCore.ContentFields/Views/HtmlField-Multiline.Edit.cshtml]: Move `OrchardCore.ContentLocalization` before `OrchardCore.ContentManagement.Metadata.Models`

File: src/OrchardCore/OrchardCore.ContentLocalization.Abstractions/CultureAspect.cs [hunk 3]
```diff
@@ -1,6 +1,6 @@
 using System.Globalization;
 
-namespace OrchardCore.Localization
+namespace OrchardCore.ContentLocalization
```
Comment (hishamco) [inline src/OrchardCore/OrchardCore.ContentLocalization.Abstractions/CultureAspect.cs]: Make sense

File: src/OrchardCore/OrchardCore/OrchardCore.csproj [hunk 4]
```diff
@@ -15,6 +15,7 @@
 
   <ItemGroup>
     <ProjectReference Include="..\OrchardCore.Abstractions\OrchardCore.Abstractions.csproj" />
+    <ProjectReference Include="..\OrchardCore.Localization.Abstractions\OrchardCore.Localization.Abstractions.csproj" />
```
Comment (hishamco) [inline src/OrchardCore/OrchardCore/OrchardCore.csproj]: Why? I expect `OrchardCore.Abstractions` may reference `OrchardCore.Localization.Abstractions`

Comment (sebastienros) [reply src/OrchardCore/OrchardCore/OrchardCore.csproj]: OrchardCore.Abstractions doesn't need it, so why do it?

**Comment by sebastienros** [comment]
> Do we really need to rename PluralizationArgument & PluralizationRuleDelegate namespace?

I actually moved the classes in a different project altogether.

**Pull request merged**